pennylane_lightning/core/src/lightning.kokkos.toml

```diff
@@ -0,0 +1,79 @@
+schema = 1
+
+[device]
+name = "lightning.kokkos"
+version = "v0.31.0"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
```

Comment:
`SparseHamiltonianObs` are also supported in all devices now.

Reply:
yes, Catalyst doesn't support `SparseHamiltonian`. Besides, `SparseHamiltonian` is not fully supported by the Lightning measurement class.